package.json

```diff
@@ -75,13 +75,14 @@
     "@babel/runtime": "^7.9.2",
     "@carbon/charts": "^0.30.5",
     "@carbon/charts-react": "^0.30.5",
-    "@carbon/colors": "10.7.0",
-    "@carbon/icons": "10.6.1",
-    "@carbon/icons-react": "10.6.1",
+    "@carbon/colors": "10.8.0",
+    "@carbon/icons": "10.9.3",
+    "@carbon/icons-react": "^10.10.0",
```

Comment:
No. We had to do this because Carbon-components-react has a carat range on their dependency  and it was resolving to this anyways. This lead to two versions of icons-react which WAS causing a problem. For some reason it doesn't respect our version listed in our deps even though it satisfies their dep